era/threat/unit.js

```diff
@@ -11,12 +11,15 @@ export class Unit {
    * @param {string} key
    * @param {string} name
    * @param {string | number} type
-   * @param {string | any[]} events
+   * @param {import("../threat/wcl.js").WCLEvent[]} events
+   * @param {import("../threat/fight.js").Fight} fight
    */
-  constructor(config, key, name, type, events) {
+  constructor(config, key, name, type, events, fight) {
```

Comment:
Added types, so that I can pass wclFight data through and know there are no bugs introduced.